src/app.tsx

```diff
@@ -9,6 +9,7 @@ import { Provider as Auth0Provider } from "./auth";
 
 import NotFound from "@reearth/components/pages/NotFound";
 import Loading from "@reearth/components/atoms/Loading";
+import NotificationBanner from "@reearth/notifications";
```

Comment:
```suggestion
import NotificationBanner from "@reearth/components/organisms/Nnotification";
```

because notifications module doesn't provide any context (provider), so this component should follow [the Component Classification Guideline](https://github.com/reearth/reearth-web/blob/main/src/components/README.md)

This component is not a page, but uses `@reearth/state`, so I think this component should be classified as a organism.